common/include/nephtys/utils/config.hpp

```diff
@@ -0,0 +1,66 @@
+//
+// Created by sztergbaum roman on 2019-04-19.
+//
+
+#pragma once
+
+#include <filesystem>
+#include <fstream>
+#include <nlohmann/json.hpp>
+#include <loguru.hpp>
+
+namespace nephtys::utils
+{
+    namespace details
+    {
+        template<typename TConfig>
+        TConfig create_configuration(std::filesystem::path &&config_path,
+                                     const std::filesystem::path &full_path) noexcept
+        {
+            TConfig config_to_export{};
+            VLOG_SCOPE_F(loguru::Verbosity_INFO, __FUNCTION__);
+            DVLOG_F(loguru::Verbosity_WARNING,
+                    "path to nephtys configuration doesn't exist, creating directories + configuration for you");
+            std::error_code ec;
+            std::filesystem::create_directories(config_path, ec);
+            if (ec) {
+                VLOG_F(loguru::Verbosity_WARNING, "creating directories failed: %s, returning default configuration",
```

Comment:
This might be a limitation of your log library, but why do you always `.c_str()` to print strings ?

Reply:
as you are pointing out the support with fmt library seems broken :/